openshift/e2e-common.sh

```diff
@@ -132,11 +144,23 @@ function run_e2e_tests(){
       local run_command="-run ^(${test_name})$"
   fi
 
-  go_test_e2e -timeout=50m -parallel=20 ./test/e2e \
+  # check for test flags
+  RUN_FLAGS=("-timeout=50m -parallel=20")
+  if [ -n "${EVENTING_TEST_FLAGS:-}" ]; then
+    RUN_FLAGS="${EVENTING_TEST_FLAGS}"
+  fi
+
+  # check for test args
+  if [ -n "${EVENTING_TEST_ARGS:-}" ]; then
+    common_opts=("${EVENTING_TEST_ARGS} --kubeconfig $KUBECONFIG")
```

Comment:
Why is KUBECONFIG passed only when EVENTING_TEST_ARGS are defined? Sounds like it should be passed as part of those EVENTING_TEST_ARGS when calling the script. Then you can avoid passing --kubeconfig on this line.